tests/plugins/test_koji_promote.py

```diff
@@ -961,13 +961,15 @@ def test_koji_promote_image_tags(self, tmpdir, os_env, additional_tags):
          None),
 
     ])
-    @pytest.mark.parametrize('has_config', [
-        True,
-        False,
+    @pytest.mark.parametrize(('has_config', 'is_autorebuild'), [
+        (True,
```

Comment:
mmm what do you mean? I can use same values for is_autorebuild from has_config, but this is more explicit